### PR TITLE
WOR-1101 Add policies from WSM to workspace details response

### PR DESCRIPTION
**Pull request by blakery**
Ticket: [WOR-1101](https://broadworkbench.atlassian.net/browse/WOR-1101)

When we get the azure coordinates of a workspace from WSM, also return the workspace's policies.

We're only changing the workspace response model, not the workspace model itself; so there's no database changes.

Once I made these changes, I was able to see the policies from WSM being retuned on a workspace when running locally.

---

**PR checklist**

- [ ] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

[WOR-1101]: https://broadworkbench.atlassian.net/browse/WOR-1101?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ It will also set up a process that will watch the local files for changes, and r
 
 See docker-rsync-local-rawls.sh for more configuration options.
 
-If Rawls successfully starts up, you can now access the Rawls Swagger page: https://local.broadinstitute.org:20443/
+If Rawls successfully starts up, you can now access the Rawls Swagger page: http://local.broadinstitute.org:20443/
```
Comment (aherbst-broad) [inline README.md]: Why this change? It can be either https or http.

File: core/src/test/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModelSpec.scala [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.dsde.rawls.model
+
+import bio.terra.workspace.model.{WsmPolicyInput, WsmPolicyPair}
+import org.scalatest.flatspec.AnyFlatSpecLike
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.mockito.MockitoSugar
+
+import java.util
+
+class WorkspaceModelSpec extends AnyFlatSpecLike with MockitoSugar with Matchers {
```
Comment (aherbst-broad) [inline core/src/test/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModelSpec.scala:10]: Is there a reason this needs to be in a separate file vs [here](https://github.com/broadinstitute/rawls/blob/develop/model/src/test/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModelSpec.scala#L11)?

**Review by aherbst-broad — APPROVED** [review]

File: core/src/test/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModelSpec.scala (hunk 2 above)
Comment (blakery) [reply core/src/test/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModelSpec.scala:10]: it uses artifacts that are only available in the core project (specifically the WSM objects)

File: README.md (hunk 1 above)
Comment (blakery) [reply README.md]: it wasn't working for me with https - I figured maybe google had tightened browser behavior or something.

Comment (blakery) [reply README.md]: I think it was still starting up, and I happened to try http when it came up 🤦

**Comment by cahrens** [comment]
I think you will need to update the Swagger API.

**Comment by cahrens** [comment]
I believe you will need to publish a new Rawls model after this and update orchestration's swagger for the workspace details response.

**Review by cahrens — APPROVED** [review]
